MovingBlocks/Terasology
# Fix for #1145 adjustable playerHeight

### Contains

Fix for #1145

### How to test

- Open Player Settings to now see two sliders for height change
- `playerHeight 2.8` in console would change height to 2.8. Only takes effect after restart of game.

### Outstanding before merging

- [x] Need to change height instantaneously without restart.
- [x] Need to find a way for saved config values in PlayerConfig.java to take effect atleast on player spawn.
- [x] Setting eye-height
